python-backend/app/api/services/document_manager_service.py

```diff
@@ -0,0 +1,56 @@
+import requests
+import base64
+
+from flask import Response
+from app.config import Config
+
+ALLOWED_DOCUMENT_CATEGORIES = ['tailings', 'permits', 'variances', 'incidents']
+
+
+class DocumentManagerService():
+    document_manager_url = f'{Config.DOCUMENT_MANAGER_URL}/document-manager'
+
+    def initializeFileUploadWithDocumentManager(self, request, mine, document_category):
```

Comment:
Any reason for not making this a class method? It doesn't look like you're leveraging anything on the instance that couldn't be used as a constant/class method instead.

Reply:
I think you are right, everything is static now.